setup.py

```diff
@@ -11,11 +11,7 @@
 
 import os
 import sys
-try:
-    from setuptools import setup, Extension
-except ImportError:
-    from distutils.core import setup, Extension
-
+from setuptools import setup, Extension
```

Comment:
pip requires setuptools if installing from source. Wheels do not use the setup.py script. Both 'bootstrapped' in 2.7 & 3.4.